tests/mir-opt/pre-codegen/checked_ops.checked_shl.PreCodegen.after.mir

```diff
@@ -35,26 +34,23 @@ fn checked_shl(_1: u32, _2: u32) -> Option<u32> {
     }
 
     bb0: {
-        StorageLive(_5);
-        StorageLive(_6);
         StorageLive(_4);
+        StorageLive(_5);
         StorageLive(_3);
-        _3 = const 31_u32;
-        _4 = BitAnd(_2, move _3);
+        _3 = BitAnd(_2, const 31_u32);
```

Comment:
Oh, thank you!  I'd been confused why these kept sticking around.

(This'll help shrink `slice::Iter::next`, too, since today it still has `_1 = 1; _3 = Offset(_2, _1);` instead of `Offset(_2, 1_usize)`.)